ansible-playbooks/ros_install_and_set_up.yml

```diff
@@ -3,62 +3,45 @@
 # 1. Check if insights-client is installed on system.
 # 2. Check if core_collect is enabled on system.
 # 3. Install pcp package on your system.
-# 4. Create configuration file for pcp
-# 5. Start and enable pmcd service
-# 6. Start and enable pmlogger service
+# 4. Configure pmlogger
+# 5. Create configuration file for pcp
+# 6. Start and enable pmcd service
+# 7. Start and enable pmlogger service
 
 - hosts: all
   become: yes
   gather_facts: True
   tasks:
+    - debug: var=ansible_distribution
+    - debug: var=ansible_distribution_major_version
+    - debug: var=ansible_architecture
```

Comment:
Do we need these variables anymore, now that we've removed the tasks that use them?

Reply:
You're right - we don't need it anymore. I was thinking of adding a banner on top saying _ROS Configurator - RHEL-8.4 x86_64_ or something, but lets remove it for now.